csharp/Samples/Microsoft.Spark.CSharp/samplesusage.md

```diff
@@ -0,0 +1,50 @@
+SparkCLRSamples.exe supports following options:
+   
```

Comment:
Because the same text file needs to be displayed at console, we may just live with this simple format for now. Otherwise, we need to add logic to remove markdown before displaying at console - which doesn't seem worth the effort now.

Sent from my Windows Phone
________________________________
From: Kaarthik Sivashanmugam<mailto:notifications@github.com>
Sent: ‎12/‎22/‎2015 12:38 PM
To: Microsoft/SparkCLR<mailto:SparkCLR@noreply.github.com>
Cc: Daniel Li (BING)<mailto:danielli@microsoft.com>
Subject: Re: [SparkCLR] load usage help text for samples from file (#185)


In csharp/Samples/Microsoft.Spark.CSharp/samplesusage.md<https://github.com/Microsoft/SparkCLR/pull/185#discussion_r48295341>:

> @@ -0,0 +1,50 @@
> +SparkCLRSamples.exe supports following options:
> +


LGTM

Consider adding bullets to the options and also format the arg names and values (for example, argument name like --temp or spark.local.dir could be in bold and value like could be in italics)

—
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/SparkCLR/pull/185/files#r48295341>.